app/helpers/orchid/application_helper.rb

```diff
@@ -116,15 +116,24 @@ def locale_link(lang_code)
     url = request.fullpath
 
     if lang_code == APP_OPTS["language_default"]
-      regex = /(?:^\/(?:#{locales}))?(\/.*|$)/
-
-      link_path = url.sub(regex, "\\1")
+      if config.relative_url_root.present?
```

Comment:
Oooh. I was looking for how to get that just the other day and I solved it in a weird and annoying way.....https://github.com/CDRH/family_letters/blob/master/app/controllers/general_override.rb#L11
so I might have to go back and mess around with this relative_url_root thing instead

Reply:
Thanks Rails folks for making it handy :D